atef/widgets/config/run_base.py

```diff
@@ -423,3 +399,43 @@ def __init__(self, *args, **kwargs):
         # modify button box labels
         self.verify_button_box.button(QDialogButtonBox.Ok).setText('Verify')
         self.verify_button_box.button(QDialogButtonBox.Cancel).setText('Reject')
+
+
+def create_tree_items(
+    data: Any,
+    parent: TreeItem,
+    prepared_file: Optional[PreparedFile] = None
+) -> None:
+    """
+    Recursively create the tree starting from the given data
+    Optionally associate prepared dataclasses with the TreeItems
+    """
+    try:
+        for cfg in data.configs:
+            if prepared_file:
+                # Grab relevant comps/configs so tree item can hold results
+                prep_configs = get_relevant_configs_comps(prepared_file, cfg)
+            else:
+                prep_configs = None
+            item = TreeItem(cfg, prepared_data=prep_configs)
+            create_tree_items(cfg, item, prepared_file=prepared_file)
+            parent.addChild(item)
+    except Exception as ex:
+        logger.debug(ex)
```

Comment:
This try/except block looks like it needs some review.

(`except Exception` with a large block of code is making me think of my scary [pmgr](https://github.com/pcdshub/pmgr/blob/master/pmgr/CfgModel.py) (`pmgr$ git grep 'except Exception' |wc -l ->     156`) migration PR 😬 - even though this is nowhere near that level of course!)